### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by cmaumet** [incf-nidash/nidmresults-spmhtml#3]
Once #1 is merged, the next step is to create a test using Matlab unittest (http://uk.mathworks.com/help/matlab/matlab_prog/author-class-based-unit-tests-in-matlab.html) that runs the viewer on the example dataset available in the `Data` folder and just checks that the program does not crash. 

@TomMaullin: each time you will create a new pull request, you will be able to run the test suite and make sure all the tests pass. This will ensure that as we update the code we do not break previously developed features. 

Once this first test is in place, we will extend the test suite to other examples of datasets in : http://neurovault.org/collections/1692/.

**Comment by TomMaullin** [comment]
Test scripts were added when the test branch was merged.
